mapeditor/mainwindow.h

```diff
@@ -98,6 +98,12 @@ private slots:
 	
 	void switchDefaultPlayer(const PlayerColor &);
 
+	void on_actionCut_triggered();
```

Comment:
is editor supposed to follow https://wiki.vcmi.eu/Coding_guidelines#Casing ?

Reply:
Those methods are automatically generated by Qt. Having exception will be easier than changing

Reply:
FYI you can always connect signals to slots manually, without using the Designer

Reply:
Manually - as with `connect` method or what's its name?
Yes, but why write boilerplate code that adds nothing to functionality when this can be done automatically behind the scenes?

Reply:
yes, connect. You can also connect to lambdas.

not saying it's a must :)